ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -1761,6 +1761,43 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
   }
 
   public isLostToFollowUp(): boolean {
-    return this.participant.data['status']=== 'CONSENT_SUSPENDED';
+    return this.participant.data['status'] === 'CONSENT_SUSPENDED';
+  }
+
+  public get isUserAllowed(): boolean {
+    return this.hasRole().allowViewSharedLearnings || this.hasRole().allowUploadRorFile;
+  }
+
+  public get isParticipantAllowed(): Observable<boolean> {
+    return this.sequencingOrdersArray$
+      .pipe(
+        map((sequencingOrders: SequencingOrder[]) => {
+          const mercuryAllow = sequencingOrders
+            .some(({pdoOrderId, sampleType}: SequencingOrder) => pdoOrderId && sampleType === 'Tumor');
+
+          const allowedStudies = ['cmi-osteo', 'cmi-lms'];
```

Comment:
@GiCharkviani - I think this has to be updated to look at realms.  This would leak between Osteo and Osteo2 since they are the same studies in DSS and different in DSM.
Since we won't get mercury data back for Osteo, this should be done in the follow-up PR as we won't encounter a case where the tab would be visible.

Reply:
@otchet-broad Elliot, the **allowedStudies** are not related to mercury data in this code anyhow. if you look at the code, on line 1779, we are reading the study name from `this.participant.data.ddp?.toLowerCase()`. I checked and both, **osteo** and **osteo2** participants have written the 'cmi-osteo' in the **ddp** field.  If Osteo study doesn't have the mercury data, then it won't be problem, but if yes - then we should read it from the selected study, instead of the **ddp** field.

**Please, check this information with @pegahtah and let me know what to do.**